From 46364999b8d4673a97d89d392c1a170295836938 Mon Sep 17 00:00:00 2001
From: User <user@example.com>
Date: Mon, 29 Jan 2024 12:47:04 +0100
Subject: [PATCH 7/8] Revert 12910e0 for the time being

---
 lib/functions/general/git.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/functions/general/git.sh b/lib/functions/general/git.sh
index ef45c0528..fbf67028c 100644
--- a/lib/functions/general/git.sh
+++ b/lib/functions/general/git.sh
@@ -55,9 +55,9 @@ function improved_git_fetch() {
 function git_ensure_safe_directory() {
 	if [[ -n "$(command -v git)" ]]; then
 		local git_dir="$1"
-		if regular_git -C "$1" -C rev-parse --git-dir > /dev/null 2>&1; then
-			display_alert "git: Marking all directories as safe, which should include" "$git_dir" "debug"
-			regular_git -C "$1" config --local --get safe.directory "$1" > /dev/null || regular_git -C "$1" config --local --add safe.directory "$1"
+		display_alert "git: Marking all directories as safe, which should include" "$git_dir" "debug"
+		if ! grep -q "directory = \*" "${HOME}/.gitconfig" 2> /dev/null; then
+			git config --global --add safe.directory "*"
 		fi
 	else
 		display_alert "git not installed" "a true wonder how you got this far without git - it will be installed for you" "warn"
-- 
2.43.0

