From e4fe0cf138a471fb6be38d32085c90d07849f2ef Mon Sep 17 00:00:00 2001
From: User <user@example.com>
Date: Sun, 15 Dec 2024 17:57:32 +0100
Subject: [PATCH] Add verbose logging to problematic jq query

---
 lib/functions/general/apt-utils.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/functions/general/apt-utils.sh b/lib/functions/general/apt-utils.sh
index c0e1bc715..540054b2a 100644
--- a/lib/functions/general/apt-utils.sh
+++ b/lib/functions/general/apt-utils.sh
@@ -37,6 +37,7 @@ function apt_find_upstream_package_version_and_download_url() {
 	if [[ "${mirror_with_slash}" != */ ]]; then
 		mirror_with_slash="${mirror_with_slash}/"
 	fi
+	set -x
 
 	declare base_down_url="http://${mirror_with_slash}pool/main/${first_letter_of_sought_package_name}/${sought_package_name}"
 
-- 
2.47.0

